src/client/app/ui.cljs

```diff
@@ -134,7 +134,7 @@
                (dom/h1 nil "Phone Numbers (click a row to edit)")
                (l/row {} (l/col {:width 2} "Phone Type") (l/col {:width 2} "Phone Number"))
                ; Show a loading message while we're waiting for the network load
-               ((domain/lag df/lazily-loaded) #(mapv ui-phone-row %) phone-numbers)))))
+               (df/lazily-loaded #(mapv ui-phone-row %) phone-numbers)))))
```

Comment:
Don't need lag for most things in Om/Untangled. Debounce is sometimes nice, but unless you're doing some kind of animation stuff you shouldn't need to worry about async garbage.